Tests.Integration.Presentation.Web/Organizations/V2/OrganizationGridApiV2Test.cs

```diff
@@ -59,25 +59,55 @@ public async Task RegularUserSeesLocalAdminChanges()
             //Local admin saves grid config
             var columns = CreateTestColumns().ToList();
             var (org, localAdminCookie) = await CreatePrerequisites();
-            await OrganizationGridConfigurationTestHelper.SaveConfigurationRequestAsync(org.Uuid,
+            await OrganizationGridTestHelper.SaveConfigurationRequestAsync(org.Uuid,
                 columns, localAdminCookie);
 
             var (_, _, userCookie) = await HttpApi.CreateUserAndLogin(CreateEmail(), OrganizationRole.User, org.Id);
 
             //User retrieves Grid Config
-            var body = await OrganizationGridConfigurationTestHelper.GetResponseBodyAsync(org.Uuid, userCookie);
+            var body = await OrganizationGridTestHelper.GetResponseBodyAsync(org.Uuid, userCookie);
             AssertColumnsMatch(columns, body.VisibleColumns.ToList());
             
 
             //Local admin deletes grid config
-            await OrganizationGridConfigurationTestHelper.SendDeleteConfigurationRequestAsync(org.Uuid, localAdminCookie);
+            await OrganizationGridTestHelper.SendDeleteConfigurationRequestAsync(org.Uuid, localAdminCookie);
 
             //Users tries to retrieve grid config after it has been deleted
             using var response = await
-                OrganizationGridConfigurationTestHelper.SendGetConfigurationRequestAsync(org.Uuid, userCookie);
+                OrganizationGridTestHelper.SendGetConfigurationRequestAsync(org.Uuid, userCookie);
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
+        [Fact]
+        public async Task LocalAdminHasConfigModificationPermissions()
+        {
+            var (org, localAdminCookie) = await CreatePrerequisites();
+            var permissions = await
+                OrganizationGridTestHelper.GetOrganizationPermissionsAsync(org.Uuid, localAdminCookie);
+            Assert.True(permissions.HasConfigModificationPermissions);
+        }
+
+        [Fact]
+        public async Task GlobalAdminHasConfigModificationPermissions()
```

Comment:
Put all of these tests into a Theory.
There you can provide the role, and an expected outcome.

For inspiration you can look here: Tests.Integration.Presentation.Web\ItSystem\ItSystemUsageTests.cs

Reply:
Hmm, not sure this is possible since each the cookies need to be properly set up with the organization.

Reply:
you can skip the user create part, the GetCookie should work for every user (I did that in other contexts and it worked, maybe it was a similar method)